README.md

```diff
@@ -95,6 +95,26 @@ gulp test:selenium
 
 is used inside the pattern-library VM and should not be used elsewhere.
 
+# Docker setup
+
+```
+docker build -t elifesciences/pattern-library .
+```
+
+(re)builds the Gulp-based Docker image, which builds all assets.
+
+```
+docker build -f Dockerfile.ui -t elifesciences/pattern-library_ui .
```

Comment:
Just thinking about naming, might be better to call this one `pattern-library` and the other `pattern-library_assets`.

Reply:
Yes I think the primitive container with gulp is not much use in itself, wasn't sure that `pattern-library` always refers to a UI originally but it seems entrenched in the workflow (you modify code, you look at a browser).